portality/view/account.py

```diff
@@ -250,6 +259,7 @@ def register():
         account = models.Account.make_account(
             username=form.w.data,
             email=form.n.data,
+            roles=[r.strip() for r in form.roles.data.split(',')]
         )
         account.set_password(form.s.data)
         account.save()
```

Comment:
Isn't this how the ```formcontext``` pre-fills the data? I wasn't aware I was doing anything novel here!